### PR TITLE
Keypoint detection

### PR DESCRIPTION
**Pull request by jakmro**

### CODE REVIEW

**Review by mkopcins — CHANGES_REQUESTED** [review]
Add example file, can be in basic tutorial.

File: lib/ex_vision/instance_segmentation/maskrcnn_resnet50_fpn_v2.ex [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    bboxes =
+      bboxes
+      |> Nx.squeeze(axes: [0])
+      |> Nx.multiply(Nx.tensor([scale_x, scale_y, scale_x, scale_y]))
+      |> Nx.round()
+      |> Nx.as_type(:s64)
+      |> Nx.to_list()
+
+    scores = scores |> Nx.squeeze(axes: [0]) |> Nx.to_list()
+    labels = labels |> Nx.squeeze(axes: [0]) |> Nx.to_list()
```
Comment (msluszniak) [inline lib/ex_vision/instance_segmentation/maskrcnn_resnet50_fpn_v2.ex:58]: Here the same code repeats. Maybe it's worth to move this code as functions to utils.

Comment (mkopcins) [reply lib/ex_vision/instance_segmentation/maskrcnn_resnet50_fpn_v2.ex:58]: @msluszniak Maybe you can add that to your PR with suggested changes

Comment (msluszniak) [reply lib/ex_vision/instance_segmentation/maskrcnn_resnet50_fpn_v2.ex]: Yes, sounds good. If there are no blockers to this PR, we can merge it and I'll rebase and add my suggestions.

**Review by mkopcins — APPROVED** [review]

File: lib/ex_vision/instance_segmentation/maskrcnn_resnet50_fpn_v2.ex (hunk 1 above)
Comment (mkopcins) [reply lib/ex_vision/instance_segmentation/maskrcnn_resnet50_fpn_v2.ex]: Merged, you can now rebase your PR

**Pull request merged**